07_errors/hsy3418/error.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import "fmt"
+
+type Error struct {
```

Comment:
Since this struct and the below `Error()` method are also a part of the exported package can you please add documentation to these as well?